newIDE/app/src/ObjectEditor/Editors/SpriteEditor/CollisionMasksEditor/index.js

```diff
@@ -178,33 +178,37 @@ const CollisionMasksEditor = (props: Props) => {
   const editorNodes =
     screenSize === 'small' ? verticalMosaicNodes : horizontalMosaicNodes;
 
+  const containerStyle = {
+    // Fixed height is required for large screens, otherwise
+    // EditorMosaic collapses to zero height
+    height: screenSize === 'small' ? '100%' : 750,
+  };
+
   if (!props.object.getAnimationsCount()) return null;
 
   const editors: { [string]: Editor } = {
     preview: {
       type: 'primary',
       noTitleBar: true,
       renderEditor: () => (
-        <div style={{ display: 'flex' }}>
-          <ImagePreview
-            resourceName={hasValidSprite ? sprite.getImageName() : ''}
-            resourcesLoader={props.resourcesLoader}
-            project={props.project}
-            onSize={setCurrentSpriteSize}
-            renderOverlay={({ imageWidth, imageHeight, imageZoomFactor }) =>
-              hasValidSprite && (
-                <CollisionMasksPreview
-                  imageWidth={imageWidth}
-                  imageHeight={imageHeight}
-                  imageZoomFactor={imageZoomFactor}
-                  isDefaultBoundingBox={sprite.isCollisionMaskAutomatic()}
-                  polygons={sprite.getCustomCollisionMask()}
-                  onPolygonsUpdated={updateCollisionMasks}
-                />
-              )
-            }
-          />
-        </div>
+        <ImagePreview
```

Comment:
Good idea to remove the extra divs, they might be the cause for some things not working with heights, so feel free to experiment removing/adding/changing these extra divs.